e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -37,25 +38,59 @@ public class PublishNodesFromFileTestTheory {
         }
 
         [Fact, PriorityOrder(2)]
-        public async Task PublishFromPublishedNodesFile() {
-            var deploy = new IoTHubPublisherDeployment(_context);
-            Assert.NotNull(deploy);
-            var result = await deploy.CreateOrUpdateLayeredDeploymentAsync();
-            _output.WriteLine("Created new layered deployment and publisher_standalone");
+        public async Task Test_CreateEdgeBaseDeployment_Expect_Success() {
+            var result = await _context.EdgeBaseDeployment.CreateOrUpdateLayeredDeploymentAsync();
```

Comment:
This check does not work. It got a successfull even though an exception was thrown.